cli/options/options.go

```diff
@@ -160,14 +167,18 @@ func GetRPCWithInvoker(gctx context.Context, ctx *cli.Context, signers []transac
 // returns an appropriate config.
 func GetConfigFromContext(ctx *cli.Context) (config.Config, error) {
 	var configFile = ctx.String("config-file")
+	var relativePath = ""
+	if argrPath := ctx.String("relative-path"); argrPath != "" {
+		relativePath = argrPath
+	}
```

Comment:
The previous variant was correct, we don't need this extra `if`. Just use the following:
```suggestion
var (
	configFile = ctx.String("config-file")
	relativePath = ctx.String("relative-path")
)
```